app/src/androidTest/java/com/github/sdp/tarjetakuna/ui/webapi/WebApiViewModelTester.kt

```diff
@@ -5,8 +5,8 @@ package com.github.sdp.tarjetakuna.ui.webapi
  */
 class WebApiViewModelTester : WebApiViewModel() {
     
-    fun getCards() {
-        super.getCardsWeb()
+    fun getRandomCard() {
+        super.getRandomCardWeb()
     }

```

Comment:
cards_by_name, cards_by_set and set_by_code in view model missing
Not sure if really needed, but i think so

Reply:
Same problem as the previous comment.